superset/dashboards/api.py

```diff
@@ -173,7 +174,7 @@ class DashboardRestApi(BaseSupersetModelRestApi):
     dashboard_get_response_schema = DashboardGetResponseSchema()
     dashboard_dataset_schema = DashboardDatasetSchema()
 
-    base_filters = [["slice", DashboardFilter, lambda: []]]
+    base_filters = [["id", DashboardAccessFilter, lambda: []]]
```

Comment:
why slice was here to begin with?

Reply:
@nytai says it's a bad copy/pasta. https://github.com/apache/superset/pull/14207#discussion_r615120219